chain-core/src/property.rs

```diff
@@ -156,7 +158,7 @@ pub trait Update {
 /// Define the Ledger side of the blockchain. This is not really on the blockchain
 /// but should be able to maintain a valid state of the overall blockchain at a given
 /// `Block`.
-pub trait Ledger<T: Transaction> {
+pub trait Ledger: Sized {
```

Comment:
It's odd that the `Sized` bound has to be included. I can't see why, but it likely does not hurt.

Reply:
It is for a following PR I will make 🤦‍♂️ sorry for including this too early.